### ISSUE TITLE
Grapher: Sparse datasets fade out

### ISSUE THREAD

**Issue by jwenner** [labrad/scalabrad-web#146]
When looking at a sparse dataset (see e.g. dataset/Jim/AliceBobRes/Cooldown151230/160103/00003/), the data points have different intensities, so for some rows the data is bright while for others it is almost impossible to see. With the old Delphi grapher, each datum had the same intensity. Note this is a problem with both version 1.0.0-M3 (on trench) and with the version on matrix-reloaded.

**Comment by maffoo** [comment]
@jwenner, please take a look now. I've added different drawing modes for 2D plots. It'll do dots and "rectfill", like the delphi grapher, as well as the new "vargrid" mode, which doesn't work for the data set you referenced, but works nicely as long all data points are drawn from a fixed set of x or y values.

**Comment by jwenner** [comment]
The different modes definitely helps, so thanks! However, even rectfill can give data which is faded out (see e.g. dataset/Jim/AliceBobRes/Cooldown151230/160208/00019/). So, I guess it's a case where you pick which one is better for a particular dataset.

**Comment by maffoo** [comment]
@jwenner, that latter dataset renders nicely in vargrid mode, so it really depends on how you take the data.

**Comment by jwenner** [comment]
Agreed. I'd say this can be closed once this fix is merged into master.

**Comment by maffoo** [comment]
> once this fix is merged into master.

That may be a while, but we'll get to it :-)

**Comment by jwenner** [comment]
@maffoo, is the version on matrix-reloaded on master branch? If so, as this is fixed on matrix-reloaded, I'd say that this issue can be closed. Thanks.

**Comment by maffoo** [comment]
@jwenner, yes, matrix-reloaded is now running 1.0.1 from master. I verified that the different 2D plotting modes work on the datasets you cited. Closing.
